pytools/pyqceff/src/pyqceff/display_qceff_table.py

```diff
@@ -0,0 +1,336 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+"""
+Script to read and pretty print QCEFF table information from a CSV file.
+"""
+
+import csv
+import sys
+from typing import Dict, List, Any
```

Comment:
python 3.8, could switch to 3.9+ but is python 3.8 still knocking about on some clusters.